codalab/apps/api/views/admin_views.py

```diff
@@ -0,0 +1,138 @@
+from rest_framework import (permissions, status, views)
```

Comment:
I just feel like having a new file for a new web seems cleaner.
Also I am querying all competitions but I am not sending everything to the client.
I am only sending the following competitions attributes:
- id
- title
- creator username & email
- start_date
- end_date
- upper_bound_max_submission_size
- max_submission_sizes

In addition to the fact that the client does not need every info on the competition some of the competitions attributes are not json serializable so they need some transformation in order to be sent. For example I could not just send the creator attribute because it is a class instance. So I created a string with the username and the email.